airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/metadata.py

```diff
@@ -209,9 +209,34 @@ def _run(self, connector: Connector) -> CheckResult:
         return self.pass_(connector=connector, message="The upgrade deadline is set to at least a week in the future")
 
 
+class CheckConnectorMaxSecondsBetweenMessagesValue(MetadataCheck):
+    name = "Certified source connector must have a value filled out for maxSecondsBetweenMessages in metadata"
+    description = "Certified source connectors must have a value filled out for `maxSecondsBetweenMessages` in metadata. This value represents the maximum number of seconds we could expect between messages for API connectors. And it's used by platform to tune connectors heartbeat timeout. The value must be set in the 'data' field in connector's `metadata.yaml` file."
+    applies_to_connector_types = ["source"]
+
+    @staticmethod
+    def check_connector_certified(connector: Connector) -> bool:
+        connector_sl_value = connector.metadata.get("ab_internal", {}).get("sl", 0)
+        return connector_sl_value >= 200
```

Comment:
You should use the `supportLevel` instead of the sl_value
```suggestion
        return  connector.metadata.get("supportLevel") == "certified"
```
I think `sl_value` is not accurate as archived connector still have the `sl_value` set to 200